src/js/components/SelectMultiple/stories/CustomMessages.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+import { Box, Text } from 'grommet';
+import { SelectMultiple } from '../SelectMultiple';
+
+const defaultOptions = [
+  'Apple',
+  'Orange',
+  'Banana',
+  'Grape',
+  'Melon',
+  'Strawberry',
+  'Kiwi',
+  'Mango',
+  'Raspberry',
+  'Rhubarb',
+];
+
+export const CustomMessages = () => {
+  const [options, setOptions] = useState(defaultOptions);
+  const [valueMultiple, setValueMultiple] = useState([]);
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" pad="large" gap="large">
+      <Text>SelectMultiple Custom messages</Text>
+      <SelectMultiple
+        value={valueMultiple}
+        placeholder="Select"
+        options={options}
+        onSearch={(text) => {
+          // The line below escapes regular expression special characters:
+          // [ \ ^ $ . | ? * + ( )
+          const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
+
+          // Create the regular expression with modified value which
+          // handles escaping special characters. Without escaping special
+          // characters, errors will appear in the console
+          const exp = new RegExp(escapedText, 'i');
+          setOptions(defaultOptions.filter((o) => exp.test(o)));
+        }}
+        onClose={() => setOptions(defaultOptions)}
+        onChange={({ value }) => {
+          setValueMultiple(value);
+        }}
+        messages={{
+          multiple: 'Multiple Selected',
```

Comment:
```suggestion
```
I think we should remove `multiple` in this example since we don't want people to actually use it and are just keeping it for backwards compatibility

Reply:
Good call. Updating now.